kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiServer.scala

```diff
@@ -105,6 +105,23 @@ object KyuubiServer extends Logging {
     val _hadoopConf = KyuubiHadoopUtils.newHadoopConf(new KyuubiConf().loadFileDefaults())
     hadoopConf = _hadoopConf
   }
+
+  private[kyuubi] def refreshUserDefaultsConf(): Unit = kyuubiServer.conf.synchronized {
+    kyuubiServer.conf.loadFileDefaults()
```

Comment:
what does this do ?

Reply:
Sorry for this mistake. legacy code from the initial version for refreshing all server configs.
Addressed.